frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -239,6 +239,7 @@ const Map: React.FC<MapProps> = ({
       }, true);
     };
     if (!isEqualWith(geoFilter, filter, compareValues)) {
+      dispatch(storeParcelDetail(null));
```

Comment:
after a search, the active parcel should be cleared. This prevents a false positive where the user might think that an active parcel is a legitimate match for a given search query.